docs/faq.md

```diff
@@ -185,6 +185,7 @@ Refer to the section [above](#how-do-i-configure-ollama-server) for how to set e
 
 - macOS: `~/.ollama/models`
 - Linux: `/usr/share/ollama/.ollama/models`
+- Arch Linux: `/var/lib/ollama/.ollama/models`
```

Comment:
This isn't a location defined by ollama so it must be set by the Arch Linux package maintainers. It's more appropriate to document it in the Arch Linux package